src/style.css

```diff
@@ -1,3 +1,79 @@
+html,
 body {
+  padding: 0;
   margin: 0;
+  width: 100%;
+}
+
+body {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+  font-style: normal;
+}
+
+:root {
+  --primary-color: #00acdc;
+}
+
+header {
+  justify-content: space-between;
+  display: flex;
+  background: #fff;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.navigation-list {
+  padding: 0;
+  margin: 0;
+  display: flex;
+  align-items: stretch;
+  height: 60px;
+  list-style: none;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.navigation-item {
+  margin-left: 20px;
```

Comment:
Be consistent with your *vertical* margins (Add only top or only bottom margin, but not both). In this case, since `.navigation-item` is a flex item, it might be more appropriate to adjust spacing using the `gap` property on the parent `.navigation-list`.